app/_src/gateway/kong-manager/auth/sessions.md

```diff
@@ -77,6 +95,21 @@ Attribute | Description
 
 {% endif_version %}
 
+{% if_version gte:3.4.x %}
+
+Attribute | Description
+----------|------------
+`cookie_name` | A name for the cookie. <br> For example, `"cookie_name":"kong_cookie"`
+`secret` | The secret used in keyed HMAC generation. Although the Session plugin's default is a random string, the `secret` _must_ be manually set for use with Kong Manager since it must be the same across all Kong workers/nodes.
+`storage` | The location where session data is stored. <br> The default value is `cookie`. It may be more secure if set to `kong`, since access to the database would be required.
```

Comment:
```suggestion
`storage` | The location where session data is stored. <br> The default value is `"cookie"`. It may be more secure if set to `"kong"`, since access to the database would be required.
```